types/xmlserializer/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "parse5": "^5.1.0"
```

Comment:
no idea why it depended on "normal" `parse5`, v7 is the first with types

Reply:
that must have been a mistake; I think the new code is correct